Packs/Core/.pack-ignore

```diff
@@ -5,4 +5,7 @@ ignore=IN145,IN124
 ignore=IN145,IN124,IN139
 
 [file:README.md]
-ignore=RM104
\ No newline at end of file
+ignore=RM104
+
+[file:impossibleTravelerGetDistance.yml]
+ignore=CJ105
```

Comment:
I think if you have the `impossibleTravelerGetDistance.yml` to `Packs/Core/Scripts/impossibleTravelerGetDistance/impossibleTravelerGetDistance.yml` it will work

Reply:
if you change*

Reply:
Doesn't work